src/template/context.js

```diff
@@ -1,5 +1,7 @@
+// Import parsing functions for each section
```

Comment:
Mb `Import function`, what do you think ?

Reply:
@svoboda-rabstvo i fixed everything. Please, review.